subprojects/gradle/build-metrics/src/main/kotlin/com/avito/android/plugin/build_metrics/BuildMetricTracker.kt

```diff
@@ -12,7 +13,11 @@ public class BuildMetricTracker(
 ) {
 
     private val node by lazy {
-        env.get().node?.take(32) ?: "unknown"
+        val env = env.get()
+        when (env.environment) {
+            is Environment.CI -> "_" // Don't need details from CI. Empty value for backward compatibility in series name
+            else -> env.node ?: "unknown"
+        }
```

Comment:
The key change in behavior

Reply:
Explain please what scenarios are affected
And why we need this change

Reply:
This is only a performance issue - excessive metrics.

We add info to a metric for debug purposes:

`...<environment>.<node>...`

Environment: `ci` | `local` | `mirakle` | `_` (unknown)
Node: username for local builds, agent name for CI builds

In CI builds any additional information about a build agent is useless. It only produces unneeded unique metrics.